### DISCUSSION TITLE
[Feature Request] Auto Message Sync on Startup

### DISCUSSION THREAD

**Discussion by LinuxinaBit** [markqvist/Reticulum#414] [Nomad Network]
This appears to cause a lot of confusion to new users, as many don't realize you have to sync to receive offline messages.
Auto sync does exist, but a lot of people aren't going to leave their computer on for 6 hours for that to happen, and/or change the delay time (if they knew to do that, they wouldn't have a problem in the first place).
That is why I propose some way to auto-sync messages as soon as possible after NomadNet starts up, preferably set to on by default.

**Comment by markqvist** [comment]
Not a bad idea. Maybe the first part of the guide text (displayed at first start) could be improved to include some information that points to the possibilities of sync settings.
